### ISSUE TITLE
Popover 508 issue

### ISSUE THREAD

**Issue by thomastighe** [GSA/sam-design-system#1027]
Popover
Please list the common component(s) involved that you're experiencing an issue with

**Expected Behavior**
1) Way to close the popover by either clicking out to closed it that is more intuitive then just clicking on the button again
2) screen reading in the popover is not allowing the reading of the content (Lists)

**Actual Behavior**
Must click open button to close
Text not reading. 

**Your Environment**
Angular Version: 12.0.8

**Ticket Number**
Please provide your JIRA ticket number that is dependent on this fix. If there is none, please create
one to validate the issue once the fix has been released. DO NOT provide the url, please ONLY
provide the ticket number.
IAE-XYZ

**Additional context**
Screenshots / recordings, minimal reproduction on stackblitz, steps to reproduce, code snippets, etc.

**Comment by cwolf10** [comment]
1) Is there a more specific behavior that has been suggested? Other than, "just not this"? Have you considered the [closeOnClickOutside](https://github.com/GSA/sam-design-system/blob/master/libs/packages/components/src/lib/popover/popover.directive.ts#L63) input? Have you considered the tooltip component?

2) Can you provide steps to reproduce this screen reader issue? I tested on SDS demo site and was able to have JAWS read the contents of the popover without issue. Were you trying to use the List Quick Key to test?

**Comment by cwolf10** [comment]
Closing this issue. Point 1 about the additional methods for closing, after speaking with Christy about this `closeOnClickOutside` will work for the initial concern. However keyboard users were a concern brought up in that discussion. As a result #1032 was done to allow for additional methods for keyboard users. This change has been released with v12.0.12.

Point 2: as I noted above, I was able to have JAWS read the content while moving through the site after the popover was opened.
